app/components/InfoTable.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+import React from 'react';
+import InfoRow from './InfoRow';
+
+function getInfoRow(item) {
+  return <InfoRow {...item}/>;
+}
+
+function getInitialState() {
+   return {
+     expanded: false
+   };
+}
+
+const unexpandedListSize = 6;
+
+class InfoTable extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = getInitialState();
+    this.handleExpandClick = this.handleExpandClick.bind(this);
+  }
+
+  handleExpandClick() {
+    this.setState({
+      expanded: !this.state.expanded
+    });
+  }
+
+  getExpandButton() {
+    if ((this.props.items.length > unexpandedListSize) && this.props.expandable) {
+      let content;
+      if (this.state.expanded) {
+        content = ['View less ', <i className='fa fa-lg fa-chevron-up'/>];
```

Comment:
All the expansion icons should have cursor: pointer styles so it's clear they are clickable.